### ISSUE TITLE
Support field prefixes

### ISSUE THREAD

**Issue by a-kokulyuk** [vbauer/jackdaw#6]
are you support prefixes for field, like "mName"?

**Comment by vbauer** [comment]
No, for what purpose? I think prefixes are generally used in languages like C or C++, but not in Java.

**Comment by a-kokulyuk** [comment]
Prefixews in java are good practice, because prefix tells you a scope of this variable, for example when you see variable with prefix "m", you than this variable is field, or var with "_" - is static variable etc.

**Comment by vbauer** [comment]
I have been developing on Java for 10 years and do not know a lot of projects with prefixes. Usually IDEs (or syntax highlighters) use different font styles and colors to show this information, for example Intellij IDEA shows:
- "static" variables with cursive font 
- "final" variables with bold font
- field variables with bold non-cursive font

**Comment by vbauer** [comment]
I need to think about this feature, maybe it could sense for some developers.

**Comment by a-kokulyuk** [comment]
Another example: when you need to find and use class field, you may type, for example, "m" and autocomplete shows you all class members.
